packages/runtime/test-runtime-utils/src/insecureUrlResolver.ts

```diff
@@ -92,11 +96,14 @@ export class InsecureUrlResolver implements IUrlResolver {
         }
     }
 
-    private resolveHelper(documentId: string) {
+    private resolveHelper(documentId: string, documentRelativePath: string) {
         const encodedTenantId = encodeURIComponent(this.tenantId);
         const encodedDocId = encodeURIComponent(documentId);
+        let documentUrl = `fluid://${new URL(this.ordererUrl).host}/${encodedTenantId}/${encodedDocId}`;
+        if (documentRelativePath  !== "") {
```

Comment:
This if condition is not needed. Adding an empty string won't change the documentUrl so you should always add the documentRelativePath.